README.md

```diff
@@ -1,3 +1,19 @@
 # ImageQuiz
 
 See [the documentation website](http://jasig.github.io/ImageQuiz).
+
+## Building
+
+Build using 
+
+```shell_session
```

Comment:
Interesting, I'm surprised linguist doesn't support all the same aliases as Rouge, since the two are supposed to be interchangeable. :confused:  :man_shrugging: